tests/helpers.js

```diff
@@ -104,6 +104,24 @@ export function validLangpackManifestJSON(extra) {
   }, extra));
 }
 
+
+export function validStaticThemeManifestJSON(extra) {
+  return JSON.stringify(Object.assign({}, {
+    manifest_version: 2,
+    name: 'My Static Theme',
+    version: '1.0',
+    theme: {
+      images: {
+        headerURL: 'weta.png',
```

Comment:
Maybe out of scope but there are Chrome properties as well (see docs/schema) which are different but we're supposed to support (addons-server doesn't right now, tbh)

Reply:
Yeah, I think that's out of scope. We are linting whatever the schema describes so I think that should be fine.